### ISSUE TITLE
Enhancement: AUC intervals table allow ratios

### ISSUE THREAD

**Issue by Gero1999** [pharmaverse/aNCA#116]
### Description
AUC intervals tables should at some point allow the user to compute their own metabolite AUC ratios or profile AUC ratios. These values should be included either in the final table as additional columns or in a new table

Bloqued until multiple analytes avaialble #3

### Definition of Done
- [ ] allow user to customize intervals to compute profile AUCs ratios (i.e, AUC_dosno2/AUC_dosno1)
- [ ] allow user to customize intervals to compute metabolite ratios AUCs (considering Molecular Weights to normalize)

**Comment by Gero1999** [comment]
Hmm something really worthy to talk about. We still need to investigate the theory in detail, but the main idea is to allow the user to fill a dataframe with columns for `analyte1`, `analyte2`, `dosnos1`, `dosnos2`,` mw1`, `mw2`. Then for each row the app will just compute the relations in the post-processing, after pk.nca.

**Comment by billdenney** [comment]
Yes, there are several subtleties, and it can be automated. We should talk about it.

**Comment by js3110** [comment]
removed matrix ratios from this issue and created a new issue #191
